scripts/sct_process_segmentation.py

```diff
@@ -205,14 +209,22 @@ def main(args):
         vert_lev = ''
     if '-r' in arguments:
         remove_temp_files = arguments['-r']
-    if '-size' in arguments:
-        smoothing_param = arguments['-size']
+    # if '-size' in arguments:
+    #     smoothing_param = arguments['-size']
     if '-vertfile' in arguments:
         fname_vertebral_labeling = arguments['-vertfile']
+    if '-perlevel' in arguments:
+        perlevel = arguments['-perlevel']
+    else:
+        perlevel = 0
     if '-v' in arguments:
         verbose = int(arguments['-v'])
     if '-z' in arguments:
         slices = arguments['-z']
+    if '-perslice' in arguments:
```

Comment:
All these blocks can be replaced by:

`arguments.get("-perslice", 0)`

The `get()` method wil try to retrieve the value by the key. If the key isn't present in the dictionary, it will return the last parameter (zero in this case) as default.

Reply:
i think your comment refers to the use of python's `argparse`, right? in this case we use `msct_parser`. It would be nice to slowly move to `argparse` thought, but this will require major refactoring which should be addressed in another PR (will affect `sct_testing`, `sct_pipeline`).